thesis-de/content-de.tex

```diff
@@ -495,38 +505,41 @@ \section{Datenhaltungssystem / API}
-einen die Abfrage der Daten an dem Datenhaltungssystem und zum anderen die der
-Ausgabe und Anzeige für den User beziehungsweise den Administrator. Eine
-Schnittstelle auf einem Webinterface dient in erster Linie der Anzeige von
-Daten, die über ein anderes System (Backend) erhalten wurden.
+Eine Schnittstelle auf einem Webinterface dient in erser Linie zur Anzeige von
+Daten, die über ein anderes System, beispielsweise aus dem Backend erhalten
+wurden. In diesem Projekt werden bewusst zwei Schnittstellen verwendet, damit
+hier die Sicherheit der Datenhaltung, wie auch die eigentliche
+Datenvisualisierung gewährleistet werden können. Im nachfolgendem werden beide
+Schnittstellen erläutert.
 
 Die erste Schnittstelle der Weboberfläche ist ein standardisierter Treiber,
 welchen man verwenden kann, um auf ein Datenhaltungssystem zuzugreifen. Dieser
-Liefert der Weboberfläche alle nötigen Funktionen und Werkzeuge zur
+liefert dem Webinterface alle nötigen Funktionen und Werkzeuge zur
 Datenextraktion, Datenmanipulation und der Rechteverwaltung auf einem
 Datenhaltungssystem. In der Ziellösung von diesem Projekt ist ein rein
 lesender Datenhaltungszugriff für die Weboberfläche vorgesehen. Dies wird durch
 den Datenhaltungssystem-Administrator gewährleistet.
 
 Neben der Verwendung eines Treibers in Verbindung mit einem
-Berechtigungskonzept, kann auch ein API-System zum Einsatz kommen. Die
-Vorteile, sowie die Rolle dieses Systems innerhalb dieses Projektes, können dem
-Abschnitt~\ref{sec:schnittstelle_datenbereitstellung} entnommen werden.
+Berechtigungskonzept, wird hier eine API-Schnittstelle zum Einsatz kommen. Die
+Vorteile, sowie die Rolle dieser Schnittstelle innerhalb dieses Projektes,
+können dem Abschnitt~\ref{sec:schnittstelle_datenbereitstellung} entnommen
+werden.
 
 Die zweite Schnittstelle dient zur Anzeige und Ausgabe der durch das
 Datenhaltungssystem erhaltenen Daten. Ohne diese Schnittstelle würden Anwender,
 welche mit der Weboberfläche arbeiten möchten, nur unstrukturierte Daten
 erhalten und könnten diese nicht zur weiteren Analyse verwenden. Der Anwender
-erhält somit einen direkten und schnellen Überblick in Form eines
-Graphens~\ref{definition_eines_graphen} oder Diagramms über die gesamten
-Datenressourcen der Infrastruktur. Diese können bei Bedarf vom Anwender oder
-dem Administrator exportiert und als Abbild festgehalten werden. Hiermit können
-anschließend Personen, welche keinen direkten Zugriff auf das System besitzen,
-Analysen oder Statistiken durchführen. Die Übertragung der Daten zwischen den
-Benutzern und dem Webserver erfolgt mit \gls{HTTPS}. Dies dient zum einem der
-Verschlüsselung der Verbindung von Client zu Weboberfläche und zurück und zum
-anderen damit die Weboberfläche selber authentifiziert werden kann.
+erhält somit einen direkten und schnellen Überblick in Form eines Graphens,
+siehe Abschnitt "Definition eines Graphen~\ref{definition_eines_graphen}" oder
```

Comment:
done
(ggf. später mal die Zeile wieder auf 80 Zeichen bringen, muss nicht in diesem PR gemacht werden)